airflow-core/src/airflow/cli/cli_config.py

```diff
@@ -233,6 +233,13 @@ def string_lower_type(val):
     action="store_true",
 )
 
+# list_dags
+ARG_LIST_LOCAL = Arg(
+    ("-l", "--local"),
+    action="store_true",
+    help="This shows local DAGs and their import errors, not what's serialized in the DB",
```

Comment:
Can we please re-phrase a bit?
```suggestion
    help="Shows local parsed DAGs and their import errors, ignores content serialized in DB",
```